### PR TITLE
FCEU/Gamepad update

### PR DESCRIPTION
**Pull request by willco007**
Moving my merge request onto a branch.

### CODE REVIEW

File: Provenance.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -1247,7 +1247,7 @@
 					};
 					1AD481B31BA350A400FDA50A = {
 						CreatedOnToolsVersion = 7.1;
-						DevelopmentTeam = 63497P68S6;
+						DevelopmentTeam = MF85NRPL4W;
```
Comment (jasarien) [inline Provenance.xcodeproj/project.pbxproj]: Please don't commit changes to the development team and/or provisioning.

File: Provenance.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -1687,7 +1687,7 @@
 					"$(SRCROOT)/PVNES/**",
 				);
 				INFOPLIST_FILE = "Provenance/Provenance-Info.plist";
-				IPHONEOS_DEPLOYMENT_TARGET = 8.0;
+				IPHONEOS_DEPLOYMENT_TARGET = 9.0;
```
Comment (jasarien) [inline Provenance.xcodeproj/project.pbxproj]: Please keep the deployment target at 8.0 for now.

File: PVSupport/PVSupport/PVEmulatorCore.h [hunk 3]
```diff
@@ -13,6 +13,8 @@
 @interface PVEmulatorCore : NSObject {
 	
 	OERingBuffer __strong **ringBuffers;
+    NSThread *_gameThread;
```
Comment (jasarien) [inline PVSupport/PVSupport/PVEmulatorCore.h]: Is there a reason for creating an instance variable for the thread?

**Comment by jasarien** [comment]
Ready to merge, pending requested changes / reasoning.

**Comment by willco007** [comment]
Sorry about that, I accidentally left that in my merge request. It is not used and can be removed.

**Pull request closed without merging**